### PR TITLE
Week 2 Assignment

### PR DESCRIPTION
**Pull request by selina-28030**
https://arabic.rt.com/photolines/1089254-%D8%A5%D9%84%D9%89-%D8%A3%D9%8A%D9%86-%D9%88%D8%B5%D9%84-%D9%81%D9%8A%D8%B1%D9%88%D8%B3-%D9%83%D9%88%D8%B1%D9%88%D9%86%D8%A7/
note: date published and modified only in application json tag

https://english.alarabiya.net/en/coronavirus/archive
note: downloading url= / ... WARNING: no html found for url= /

Wasn't able to finish 6 urls, xpaths are commented out

### CODE REVIEW

File: metahtml/timestamp.py [hunk 1]
```diff
@@ -383,7 +383,7 @@ def get_timestamp_published(html, url, **kwargs):
         ( 'stripes.com',                    '//span[@class="published_date"]' ),
         ( 'thediplomat.com',                '//span[@itemprop="datePublished"]' ),
         ( 'theintercept.com',               '//span[@class="PostByline-date"]' ),
-        ( 'theworldnews.net',               '//span[@class="news__time"]/text()'),
+        #( 'theworldnews.net',               '//span[@class="news__time"]/text()'),
```
Comment (mikeizbicki) [inline metahtml/timestamp.py:386]: In general, you shouldn't have commented out code within a project.  It should just be deleted.

File: tests/golden.csv [hunk 2]
```diff
@@ -753,7 +1138,11 @@ Mike Izbicki,seattletimes.com,2020-05-30,https://www.seattletimes.com/nation-wor
 Mike Izbicki,semana.com,2020-05-31,https://www.semana.com/,FALSE,,,,
 Mike Izbicki,semana.com,2020-05-31,https://www.semana.com/mundo/articulo/,FALSE,,,,
 Mike Izbicki,semana.com,2020-05-31,https://www.semana.com/mundo/articulo/en-grave-peligro-hermetismo-sobre-estado-de-salud-de-kim-jong-un-tras-cirugia/664851,TRUE,es-co,2020-04-20 22:29:00,2020-04-21 23:01:47,¿En grave peligro? Hermetismo sobre estado de salud de Kim Jong-un tras cirugía
-,sfweekly.com,2020-06-06,https://www.sfweekly.com/,FALSE,,,,
```
Comment (mikeizbicki) [inline tests/golden.csv:756]: Is there a reason this line is deleted?

**Comment by mikeizbicki** [comment]
The substantive changes of this pull request look good, but there are some minor git issues that I'd like you to fix before I merge the pull request.  In particular: 

1. I need the pull request to contain only a single commit, whereas yours currently contains 7 different commits.  Probably the easiest way for you to make this happen is to refork my repo and then apply your changes again.  If I were doing this, however, I would do what's called "rebasing" the repo to "squash" all of these commits into a single commit.  [This stackexchange post](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) has instructions.

1. Your final commit cannot include the `.ds_store` or the `.ipnyb` files.  You will have to delete those from your commit before issuing the new pull request.  In the future, do not run `git add .` or `git add -A`.  Instead, explicitly specify which files you want added.  The instructions contain the correct git add command which will only add the correct files.

File: tests/golden.csv (hunk 2 above)
Comment (selina-28030) [reply tests/golden.csv:756]: On my version, that row has just been moved down after inserting the new urls from line 1141 to line 1145.

Comment (mikeizbicki) [reply tests/golden.csv:756]: Ahh, I see.  You're right.  Ignore my previous comment :)

**Pull request closed without merging**